verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -140,9 +140,13 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
         return {
             demographics: {
                 sex: rowData.sex,
-                age: {
-                    start: rowData.age,
-                },
+                ageRange:
+                    rowData.age[0] !== null || rowData.age[1] !== null
```

Comment:
Is this check needed? Does the following work?
```ts
ageRange: {
    start: rowData.age[0] ?? undefined;
    end: rowDate.age[1] ?? undefined;
}
```

Reply:
ah yeah I forgot about the ?? :) done.